### PR TITLE
feat: Create hub agent

### LINKED ISSUE #1: Create hub agent web
1- Add hub agent web app (https://vuejs.org/)
2- Start hub agent rest server

### PR DESCRIPTION
**Pull request by fqutishat**
closes #1

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: cmd/mediator/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# mediator
```
Comment (troyronda) [inline cmd/mediator/README.md]: isn't the hub-agent ui project?
why `mediator`

Comment (fqutishat) [reply cmd/mediator/README.md]: should I call it cmd/ui/........

File: test/bdd/fixtures/agent-rest/.env [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# variables. See docker compose documentation.
+
+ARCH=amd64
+
+# Agent configurations
+AGENT_REST_IMAGE=trustbloc/aries-agent-rest
+AGENT_REST_IMAGE_TAG=0.1.1-snapshot-2cccd53
+
+# Mediator agent configurations
+MEDIATOR_HOST=0.0.0.0
```
Comment (fqutishat) [inline test/bdd/fixtures/agent-rest/.env]: @troyronda  should I change it to HUB_HOST

Comment (troyronda) [reply test/bdd/fixtures/agent-rest/.env]: HUB_AGENT_HOST I guess.

File: cmd/mediator/README.md (hunk 1 above)
Comment (troyronda) [reply cmd/mediator/README.md]: there are a few options I guess:

- ui
- www
- hub-agent-ui
- hub-agent-www
- dashboard-ui
- etc.

**Comment by troyronda** [comment]
- I'd rather not commit static/binary assets into the repo, particularly a logo we will not use.
  - [git-lfs](https://github.blog/2015-10-14-managing-large-files-with-git-lfs/) is more appropriate for static/binary assets.

File: test/bdd/fixtures/agent-rest/.env (hunk 2 above)
Comment (fqutishat) [reply test/bdd/fixtures/agent-rest/.env]: Done

File: cmd/mediator/README.md (hunk 1 above)
Comment (fqutishat) [reply cmd/mediator/README.md]: Done change it to ui

**Comment by fqutishat** [comment]
> * I'd rather not commit static/binary assets into the repo, particularly a logo we will not use.
>   
>   * [git-lfs](https://github.blog/2015-10-14-managing-large-files-with-git-lfs/) is more appropriate for static/binary assets.

done

**Pull request merged**